### DISCUSSION TITLE
【作业提交】Python Plonk #1

### DISCUSSION THREAD

**Discussion by ShirleneLiu** [zkp-co-learning/zkp-co-learn#83] [Show and tell]
ZKP 进阶共学！
把你的作业 repo 提交到评论区吧~

1、office hour 的时候可以直接打开某条作业点评
2、便于从 participates 区域统计大家的作业完成情况

**Comment by pochenai** [comment]
https://github.com/dajuguan/MIT-ZKIAP-2023/blob/main/module_1.md
